docs/deployment.md

```diff
@@ -141,3 +141,22 @@ If you need to specify the secrets to be used for this environment:
   - RELEASES_SECRETS :: the releases.exs file containing secrets specific to the alpha environment.
 
 
+### Release guide format
+
+1. Release to production and Alpha should at least be done once per week if there are issues/features that has been worked on.
+
+2. For small tweaks like , fixing bugs, dependabot changes, updating a single mobility page a `double dot release` should be done. That is the last digit is increased by one i.e `(v0.12.3 to v0.12.4)`.
+
+3. When a new feature has been worked on or a bunch of mobility pages has been updated, a `single dot realease` should be done. That is, increasing the middle digit by one i.e`(v0.13.0 to v0.14.0)` .
+
+4. Making a release, involves the following cycle:
+     
+     * Sanity check development is stable/working.
+     * Making release to Alpha.
+     * Incrementing development release number (so any future work goes to "next double dot".
+     * Sanity check alpha works.
+     * If alpha has issues, fix in dev and start with first bullet again but on next release number (ie fix is next double dot digit from the broken release).
+     * if alpha passes sanity check, make a release to production.
+     * Sanity check production works (if not, fix in development and go through cycle again)
+
+    
```

Comment:
Can we add the instance names to staging, alpha, and production and also their static IP addresses. So that one can easily know the instance they're working with at each level.